pkg/nimble/contrib/nimble_riot.c

```diff
@@ -119,4 +126,13 @@ void nimble_riot_init(void)
 #ifdef MODULE_NIMBLE_SVC_IPSS
     ble_svc_ipss_init();
 #endif
+
+#if defined(MODULE_NIMBLE_AUTOCONN) && !defined(MODULE_NIMBLE_AUTOCONN_NOAUTOINIT)
+    ble_gatts_start();
+    /* CAUTION: this must be called after nimble_netif_init() and also only
```

Comment:
Can this comment be added under the function documentation?

Reply:
done